data/uhc_pack/functions/pre_generation/stage/generate.mcfunction

```diff
@@ -12,8 +12,8 @@ scoreboard players reset ChunkSize uhcPG
 
 scoreboard players operation Total uhcPG *= Total uhcPG
 
-bossbar add uhc_pack:pregen {"text":"Generating...","color":"white"}
-bossbar set uhc_pack:pregen color green
-bossbar set uhc_pack:pregen players @a
-bossbar set uhc_pack:pregen style notched_10
 execute store result bossbar uhc_pack:pregen max run scoreboard players get Total uhcPG
+bossbar set uhc_pack:pregen value 0
+bossbar set uhc_pack:pregen visible true
```

Comment:
I believe you'll need to move the `bossbar set uhc_pack:pregen players @a` back to here. Otherwise if the datapack is loaded when no one is on the server, no one will see the bossbar